js/store.js

```diff
@@ -192,35 +205,26 @@ const store = {
     }
   },
 
-  getThirdParties(hostname) {
-    if (!hostname) {
-      throw new Error('getThirdParties requires a valid hostname argument');
-    }
-
-    const firstParty = this.getWebsite(hostname);
-    if ('thirdPartyHostnames' in firstParty) {
-      return firstParty.thirdPartyHostnames;
-    }
-
-    return null;
-  },
-
   setWebsite(hostname, data) {
     const websites = clone(this._websites);
```

Comment:
When you say...

> make sure that setFirstParty blocks the thread of the capture code otherwise this will happen

What would that implementation look like? This is not the same as adding or removing `await` per our other discussion [here](https://github.com/biancadanforth/lightbeam-we/commit/5abf415cc1042dc08373d8db9b44c04765c975f5#commitcomment-23517430), is it?